Fixed elements spacing

salman-farooqi opened the pull request:
<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**
bugfix, extra spacing fixed (If further changes required, tell me please)

**Issue Number:**
Fixes #456 

**Snapshot:**
![spacing fixed](https://user-images.githubusercontent.com/127323880/224533925-55bf64d1-cd18-4b5e-9eb0-e126834bff0b.JPG)

**Summary**
Headings were given a lot of margins. By reducing the extra spaces of the entire website, it gives a decent look now.
(If further changes required, tell me please)

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**
Yes

salman-farooqi committed:
fixed extra spacing between elements

salman-farooqi committed:
fixed extra spacing between elements

palisadoes reviewed (changes requested):
- This is a CSS change. Yarn.lock is not required. Please remove it from the PR.
- Also, update to the latest version of docusaurus, that's probably why there is a variance in yarn.lock

salman-farooqi commented:
> * This is a CSS change. Yarn.lock is not required. Please remove it from the PR.
> * Also, update to the latest version of docusaurus, that's probably why there is a variance in yarn.lock

Ok, thank you, I'm doing it.

salman-farooqi committed:
Merge branch 'PalisadoesFoundation:develop' into fixed-elements-spacing

salman-farooqi committed:
Deleted yarn.lock

Deleted in yarn.lock in context of this below...
This is a CSS change. Yarn.lock is not required. Please remove it from the PR.
Also, update to the latest version of docusaurus, that's probably why there is a variance in yarn.lock

salman-farooqi commented:
I'm really sorry, couldn't find a way to merge new changes to this PR, that's why had to create a new PR #469.
